test/maxAge.js

```diff
@@ -326,6 +322,26 @@ describe('Max-Age', () => {
     await cache.drop(bodySegment);
   });
 
+  it('does not store if no-store', async () => {
+    const cache = createCache();
+
+    api.get('/').reply(200, defaultResponse, { 'cache-control': 'no-store' });
+
+    await requestWithCache(cache);
+    const cached = await cache.get(bodySegment);
+    assert.equal(cached, null);
```

Comment:
We're importing `chai` assert so we can use more expressive assertion functions. In this case, `assert.isNull`

Reply:
👍 updated https://github.com/bbc/http-transport-cache/pull/20/commits/f515c1773dec7949ddfd2b426199087f6849d516